CODEOWNERS

```diff
@@ -372,6 +377,7 @@
 /drivers/timer/*rv32m1_lptmr*             @mbolivar
 /drivers/timer/*nrf_rtc*                  @anangl
 /drivers/timer/*hpet*                     @dcpleung
+/drivers/timer/*rts5912*                  @JasonLin-RealTek
```

Comment:
this and others seem to be all in the board commit -> wrong

Reply:
Fix, moved them to where they should be.

Reply:
stil here, btw you'd be better off ignoring this file entirely and just use the `MAINTAINERS.yml` entry